styles.css

```diff
@@ -277,3 +276,80 @@ ul {
   color: #253858;
   margin-left: 1rem;
 }
+
+#reach-out {
+  align-items: center;
+  background-image: url(/images/about-image/contact-form-shapes-mobile.svg);
+  margin: 0%;
+  width: 100%;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-color: #6070ff;
+  background-position: right;
+  border-radius: 15% 0 0 0;
+}
+
+.contact-me {
+  color: #fff;
+  text-align: center;
+  margin-top: 85px;
+}
+
+.cont-input {
+  font-family: var(--main-font-family);
+  text-align: center;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+form {
+  margin-right: 45px;
+  display: flex;
+  flex-direction: column;
+}
+
+.username {
+  width: 100%;
+  height: 48px;
+  padding: 5px;
```

Comment:
- [ ] Great job with your design 👍 . I can see you and your partner put in a lot of effort. Kindly make sure your design looks as close as possible to the Figma design as it is best practice. This issue is caused by the padding in which you used on your `main` tag.

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/177372090-f89ac473-4653-458c-9da9-0728a09b5e69.png)  |  ![image](https://user-images.githubusercontent.com/74814780/177372151-fc44cecc-392a-4f88-8114-73e141de95c4.png)